LICENSE.md

```diff
@@ -28,6 +28,22 @@ License: [The Apache Software License, Version 2.0](http://www.apache.org/licens
 
 ===========================================================================
 
+Mapbox Maps Android uses portions of the Android DataStore.
```

Comment:
@ank27 do ew have understanding where does this come from? and if it's expected. Datastore / coroutines - is this what we need in SDK?

Reply:
Its coming from data store implementation from common's settings api for i18n. its expected and we do need these.

datastore internally uses coroutines, which means this is also required.

Reply:
thanks for clarification, that's good it's expected although I'm curious how it will affect binary size, we need to keep an eye on that